public/legacy/modules/AOS_Products/AOS_Products.php

```diff
@@ -64,10 +64,12 @@ public function save($check_notify = false)
         require_once('include/upload_file.php');
         $GLOBALS['log']->debug('UPLOADING PRODUCT IMAGE');
 
-        $imageFileName = $_FILES['uploadimage']['name'] ?? '';
-        if (!has_valid_image_extension('AOS_Products Uploaded image file: ' . $imageFileName , $imageFileName)) {
-            LoggerManager::getLogger()->fatal("AOS_Products save - Invalid image file ext : '$imageFileName'.");
-            throw new RuntimeException('Invalid request');
+        if(!empty($_FILES['uploadimage'])){
```

Comment:
Hi @bralandealmeida, just one more thing sorry.

We need to check for `!empty($_FILES['uploadimage']['name'])` instead of just `!empty($_FILES['uploadimage'])`. 

We were still getting the error with just `!empty($_FILES['uploadimage']`